experiment_manager.py

```diff
@@ -8,9 +8,12 @@
 from dataclasses import dataclass
 import Labber
 import itertools as iter
+
+from tictoc import tic, toc
+
 from general_utils import enumerated_product
 
-sys.path.append(os.path.abspath(r"G:\My Drive\guy PHD folder\util"))
+#sys.path.append(os.path.abspath(r"G:\My Drive\guy PHD folder\util"))
 import Labber_util as lu
```

Comment:
Reformat imports.
Use `Ctl`+`Alt`+`L` to reformat the file and `Ctl`+`Alt`+`M` to optimize imports.

Reply:
fixed (but ctl+alt+M= doesn't work in my version)